### PR TITLE
test: GtfsTransferTableLoader class

### PR DESCRIPTION
**Pull request by lionel-nj**
closes #570 

**Summary:**

This PR provides additional unit test for GtfsTransferTableLoader class

**Expected behavior:** 

No code change, tests should pass

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `gradle test` to make sure you didn't break anything
- [x] Format the title like "feat: -new feature short description-" (PR title must follow the [conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/))
- [x] Linked all relevant issues
- ~[ ] Include screenshot(s) showing how this pull request works and fixes the issue(s)~

### CODE REVIEW

File: main/src/test/java/org/mobilitydata/gtfsvalidator/table/GtfsTransferTableLoaderTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mobilitydata.gtfsvalidator.input.GtfsFeedName;
+import org.mobilitydata.gtfsvalidator.notice.EmptyFileNotice;
+import org.mobilitydata.gtfsvalidator.notice.MissingRequiredFieldError;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.validator.ValidatorLoader;
+
+/** Runs GtfsFareAttributeContainer on test CSV data. */
```
Comment (maximearmstrong) [inline main/src/test/java/org/mobilitydata/gtfsvalidator/table/GtfsTransferTableLoaderTest.java:33]: I think it should be `GtfsTransferTableContainer` instead

File: main/src/test/java/org/mobilitydata/gtfsvalidator/table/GtfsTransferTableLoaderTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+public class GtfsTransferTableLoaderTest {
+  private static final GtfsFeedName FEED_NAME = GtfsFeedName.parseString("au-sydney-buses");
+
+  @Test
+  public void validFileShouldNotGenerateNotice() throws IOException {
+    ValidatorLoader validatorLoader = new ValidatorLoader();
+    Reader reader =
+        new StringReader(
+            "from_stop_id,to_stop_id,transfer_type"
+                + System.lineSeparator()
```
Comment (maximearmstrong) [inline main/src/test/java/org/mobilitydata/gtfsvalidator/table/GtfsTransferTableLoaderTest.java:44]: I have seen Alexej mentioning we could change `System.lineSeparator()` for `"\n"` in another PR. Do you think we can change it here as well?

File: main/src/test/java/org/mobilitydata/gtfsvalidator/table/GtfsTransferTableLoaderTest.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    assertThat(transfer.transferType()).isEqualTo(GtfsTransferType.MINIMUM_TIME);
+  }
+
+  @Test
+  public void missingRequiredFieldShouldGenerateNotice() throws IOException {
+    ValidatorLoader validatorLoader = new ValidatorLoader();
+    Reader reader =
+        new StringReader(
+            "from_stop_id,to_stop_id,transfer_type"
+                + System.lineSeparator()
```
Comment (maximearmstrong) [inline main/src/test/java/org/mobilitydata/gtfsvalidator/table/GtfsTransferTableLoaderTest.java:68]: Same as above

**Review by maximearmstrong — CHANGES_REQUESTED** [review]

File: main/src/test/java/org/mobilitydata/gtfsvalidator/table/GtfsTransferTableLoaderTest.java (hunk 2 above)
Comment (lionel-nj) [reply main/src/test/java/org/mobilitydata/gtfsvalidator/table/GtfsTransferTableLoaderTest.java:44]: Yes! Thanks for reminding me that. I'll update this PR as well as every PR related to Gtfss entity loaders and containers.

File: main/src/test/java/org/mobilitydata/gtfsvalidator/table/GtfsTransferTableLoaderTest.java (hunk 3 above)
Comment (lionel-nj) [reply main/src/test/java/org/mobilitydata/gtfsvalidator/table/GtfsTransferTableLoaderTest.java:68]: https://github.com/MobilityData/gtfs-validator/pull/627#discussion_r564062201

**Comment by lionel-nj** [comment]
Closing as a trivial test to avoid artificially inflating the code base.

**Pull request closed without merging**